deploy/kubernetes/base/node.yaml

```diff
@@ -17,7 +17,6 @@ spec:
       nodeSelector:
         kubernetes.io/os: linux
         kubernetes.io/arch: amd64
-      hostNetwork: true
```

Comment:
Please update the helm chart as well to make them consistent: https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/master/helm

Reply:
Actually similar change has been picked up in https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/159. 

Merged from master.